### PR TITLE
Remove Doctrine type from Sqlite default parameter

### PR DESCRIPTION
**Pull request by GwendolenLynch**
|      Q       |   A
|------------- | -----------
| Type         | bug
| BC Break     | no
| Fixed issues | #3220

#### Summary

Unsure if this is the best place to do this, but putting issue & PR in motion to at least start the discussion.

### CODE REVIEW

**Review by Majkl578 — DISMISSED** [review]
Please retarget this PR to master.

**Comment by GawainLynch** [comment]
> Please retarget this PR to master.

Done :+1:

**Comment by JarJak** [comment]
any news on this one @Majkl578 @GawainLynch ?

**Comment by SenseException** [comment]
@Majkl578 Should also a test be added that covers the (re)set behaviour of the default column value, that is described in the issue?

File: lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php [hunk 1]
```diff
@@ -480,4 +480,13 @@ private function parseColumnCommentFromSQL(string $column, string $sql) : ?strin
 
         return '' === $comment ? null : $comment;
     }
+
+    private function removeDoctrineTypeFromDefault(?string $default) : ?string
+    {
+        if (preg_match('/^(.*)(--\(DC2Type:.*\))/', $default, $match)) {
```
Comment (greg0ire) [inline lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php]: Maybe make the second group non-capturing?
```suggestion
        if (preg_match('/^(.*)(?:--\(DC2Type:.*\))/', $default, $match)) {
```

**Comment by zorn-v** [comment]
Any progress ?

**Comment by zorn-v** [comment]
Guys it is very terrible that mogrations generate a lot of crap because of this for sqlite DB.

**Pull request closed without merging**